frontend/src/components/AddRecipe.tsx

```diff
@@ -145,46 +150,51 @@ export default function AddRecipe(props: AddRecipeProps) {
             <Link to="/">
                 <img src="/icons8-zurück-48.png" alt="Back" />
             </Link>
-            <div className="recipe-card">
+            <div className="div_description">
+                <div className="recipe-title">
                 <h1>Neues Rezept</h1>
+                </div>
                 <form>
                     <h2>
```

Comment:
Warum ist das alles in ein `<h2>`?
Wenn du nur das Styling von h2 haben möchtest, dann solltest du über `className` eine Klasse vergeben und im CSS mit `font-size`, `font-weight` und `margin` hantieren.

Reply:
weil ich mit den Klassen leider erhebliche Probleme hatte und es nicht so funktionierte, wie ich es gern gehabt hätte. Ich gehe absolut d'accord, dass man dies darüber steuern sollte, aber nach ewigen tüfteln bin ich Back to the roots und habe es meinen Fähigkeiten entsprechend minimalistisch belassen :)